common/innovation/innovation.js

```diff
@@ -1909,7 +1909,7 @@ Innovation.prototype.getAchievementsByPlayer = function(player) {
     }
   }
 
-  const flags = this
+  const flagsTemp = this
```

Comment:
This shouldn't be assigned at all, it looks like. Better to remove `const flags` than to rename it like this.